EpicLoot/Patching/FilePatching.cs

```diff
@@ -2,10 +2,12 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Runtime.CompilerServices;
```

Comment:
What is this using for?

Reply:
This is probably an artifact.  Let me see if I can remove it